### ISSUE TITLE
gatsby-plugin-image@1.1.0 is outdated

### ISSUE THREAD

**Issue by axelra82** [GraphCMS/gatsby-source-graphcms#194]
Please consider adding `gatsby-plugin-image@1.x` to peerDependencies. Current 1.1.0 is outdated.

```
npm ERR! Found: gatsby-plugin-image@1.5.0
npm ERR! node_modules/gatsby-plugin-image
npm ERR!   gatsby-plugin-image@"^1.5.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby-plugin-image@"1.1.0" from gatsby-source-graphcms@2.5.2
npm ERR! node_modules/gatsby-source-graphcms
npm ERR!   gatsby-source-graphcms@"*" from the root project
```

**Comment by notrab** [comment]
@axelra82 happy to accept a PR 😅

**Comment by jpedroschmitz** [comment]
Fixed it on #206!

As soon as we publish it as stable, I'll let you know 👊🏻
